src/core/reference/include/openvino/reference/string_tensor_pack.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+#include "openvino/core/shape.hpp"
+
+namespace ov {
+namespace reference {
+template <typename T_idx>
+void string_tensor_pack(const T_idx* begins,
+                        const T_idx* ends,
+                        const uint8_t* symbols,
+                        std::string* out,
+                        const int64_t symbol_count,
+                        const int64_t string_count) {
+    const char* chars = reinterpret_cast<const char*>(symbols);
+    for (int64_t i = 0; i < string_count; ++i) {
+        out[i].assign(chars + begins[i], chars + ends[i]);
+    }
```

Comment:
`symbol_count` is not used in this function

Reply:
Removed, good catch :)